airbyte-webapp/src/views/Connector/ConnectorForm/useBuildForm.tsx

```diff
@@ -54,19 +70,18 @@ interface BuildUiWidgetsContextHook {
 
 export const useBuildUiWidgetsContext = (
   formFields: FormBlock[] | FormBlock,
-  formValues: ConnectorFormValues,
-  uiOverrides?: WidgetConfigMap
+  formValues: ConnectorFormValues
 ): BuildUiWidgetsContextHook => {
-  const [overriddenWidgetState, setUiWidgetsInfo] = useState<WidgetConfigMap>(uiOverrides ?? {});
+  const [overriddenWidgetState, setUiWidgetsInfo] = useState<WidgetConfigMap>({});
 
   // As schema is dynamic, it is possible, that new updated values, will differ from one stored.
   const mergedState = useMemo(
     () =>
       merge(
         buildPathInitialState(Array.isArray(formFields) ? formFields : [formFields], formValues),
-        merge(overriddenWidgetState, uiOverrides)
+        merge(overriddenWidgetState)
```

Comment:
```suggestion
        overriddenWidgetState
```
This `merge()` call is now unnecessary